src/app/rundown-view/components/segment/segment.component.html

```diff
@@ -6,7 +6,7 @@
   <div [cdkContextMenuTriggerFor]="contextMenu" class="c-segment__header">
     <div class="c-segment__header-toolbar">
       <span>123</span>
```

Comment:
Wasn't this removed?

Reply:
Might have been, i had no idea this was there. It is removed in SOF-1680!